docs/check-redirects/check-redirects.js

```diff
@@ -1,4 +1,3 @@
-const yaml = require('yaml');
```

Comment:
Unused, plus we don't have `yaml` installed.

Reply:
@ptgott Could you verify that?

Reply:
I guess the same goes for the memfs package that was added to package.json to satisfy the script in docs I presume.

Reply:
To me it feels like `docs/check-redirects` should have its own package.json that's not necessarily a part of the workspace dictated by the root `package.json`.

Reply:
Yeah, that would be probably better, we do the same for `e2e/package.json`.